scripts/generate-doc.sh

```diff
@@ -32,7 +32,7 @@
 
 set -euo pipefail
 
-TF_VERSION="${TF_VERSION:-"1.6"}" # TF version to use when running tfplugindocs. Default: 1.6.6
+TF_VERSION="${TF_VERSION:-"1.6"}" # TF version to use when running tfplugindocs. Default: 1.6
```

Comment:
is this the same as 1.6.x?

Reply:
unfortunately it does not work

```bash
downloading Terraform CLI binary version from releases.hashicorp.com: 1.6.x
panic: Malformed version: 1.6.x

goroutine 1 [running]:
github.com/hashicorp/go-version.Must(...)
	/Users/andrea.angiolillo/go/pkg/mod/github.com/hashicorp/go-version@v1.6.0/version.go:102
```

Reply:
what does 1.6 mean then?

Reply:
I believe the latest 1.6 version although the log does not specify it 😞